src/vs/workbench/browser/parts/editor/editorStatus.ts

```diff
@@ -792,7 +793,7 @@ export class EditorStatus extends Disposable implements IWorkbenchContribution {
 		this.updateState(info);
 	}
 
-	private onScreenReaderModeChange(editorWidget: ICodeEditor | undefined): void {
+	private onScreenReaderModeChange(editorWidget: ICodeEditor | undefined, accessiblitySupportEnabled?: boolean): void {
```

Comment:
sorry to nitpick here but why do we even add this as a parameter to the function? we have the accessibility service in the function and we can call it ourselves?

Reply:
I wanted to handle this in one place, but makes sense to just call it there. I'll go with that 👍🏼

Reply:
I did mean it in one place, inside the function, but you don't need it passed to you since you have access to the service in there. You already make a call on line 804, so I thought you could do the same.